src/app/components/indicators.jsx

```diff
@@ -0,0 +1,12 @@
+const Indicators = () => {
+  return (
+    <>
+      <div className="relative w-[70px] h-[70px] border-4 animate-spin-slow border-black border-dashed rounded-full"></div>
+      <div className="absolute animate-spin-slower flex items-center justify-center border-2 border-dashed rounded-full top-[7px] left-[7px] w-14 h-14 border-slate-950">
+        <div className="w-[44px] h-[44px] bg-black rounded-full " />
+      </div>
+    </>
```

Comment:
Please check the original indicator and the motion. 
- On the original project the outer circle has an animation of decreasing and increasing in size as well. 
- I like the color choice on your version, but keep in mind that the general theme of the project is going to be darker so try to imagine it but that can be done later.
- don't also forget that it's clickable component, but that can also be managed later

Reply:
Thank you for your feedback @NatiDeme, 
- I have added the scaling animation, the circle has a decreasing and increasing size animation.
- I have added a dynamic color for the Indicator component we can pass any color for the button.
- The clickable components can be managed later on when we add them to the map